branchinfo.txt

```diff
@@ -4,6 +4,6 @@
 MAJOR_VERSION=1
 MINOR_VERSION=0
 PATCH_VERSION=0
-RELEASE_SUFFIX=rc3
-CHANNEL=preview
+RELEASE_SUFFIX=rc4
+CHANNEL=master
 BRANCH_NAME=master
```

Comment:
There will need to be a reckoning here someday.  "CHANNEL" and "BRANCH_NAME" should really be combined.  Can you do it here and now?